### PR TITLE
Few more Java 17 syntax improvements.

### PR DESCRIPTION
**Pull request by bodiam**

### CODE REVIEW

File: src/main/java/net/datafaker/idnumbers/pt/br/IdNumberGeneratorPtBrUtil.java [hunk 1]
```diff
@@ -140,10 +140,9 @@ public static int digit(int verifyingDigit) {
 
     private static String leftPad(char pad, int length, String string) {
         StringBuilder appender = new StringBuilder();
-        for (int i = 0; i < length - string.length(); i++) {
-            appender.append(pad);
-        }
-        return appender.append(string).toString();
+        appender.append(String.valueOf(pad).repeat(Math.max(0, length - string.length())));
+        appender.append(string);
+        return appender.toString();
```
Comment (snuyanzin) [inline src/main/java/net/datafaker/idnumbers/pt/br/IdNumberGeneratorPtBrUtil.java:145]: no need for `StringBuilder` here

Comment (bodiam) [reply src/main/java/net/datafaker/idnumbers/pt/br/IdNumberGeneratorPtBrUtil.java:145]: Why not? What's the alternative?

Comment (snuyanzin) [reply src/main/java/net/datafaker/idnumbers/pt/br/IdNumberGeneratorPtBrUtil.java:145]: String.valueOf(pad).repeat(Math.max(0, length - string.length())) + string;

Comment (bodiam) [reply src/main/java/net/datafaker/idnumbers/pt/br/IdNumberGeneratorPtBrUtil.java:145]: Ah, like that. Yes, yes, can do.

File: src/main/java/net/datafaker/transformations/sql/SqlTransformer.java [hunk 2]
```diff
@@ -484,7 +484,6 @@ public String getValue(Case caze) {
                 case UPPERCASE -> upperCaseValue;
                 case LOWERCASE -> lowerCaseValue;
                 case CAPITAL -> capitalValue;
-                default -> throw new IllegalArgumentException("Unknown case " + caze);
```
Comment (snuyanzin) [inline src/main/java/net/datafaker/transformations/sql/SqlTransformer.java:487]: what happens if someone adds a new value to enum and forgets to update this `switch`?
With `default` exception the problem could be detected immediately. 
How to detect it without `default`?

Comment (bodiam) [reply src/main/java/net/datafaker/transformations/sql/SqlTransformer.java:487]: Then it won't compile:

image

Comment (snuyanzin) [reply src/main/java/net/datafaker/transformations/sql/SqlTransformer.java:487]: ok then

**Review by snuyanzin — APPROVED** [review]

File: src/test/java/net/datafaker/providers/base/AustraliaTest.java [hunk 3]
```diff
@@ -7,10 +7,10 @@ class AustraliaTest extends BaseFakerTest<BaseFaker> {
 
     @Override
     protected Collection<TestSpec> providerListTest() {
-        Australia astralia = faker.australia();
-        return List.of(TestSpec.of(astralia::locations, "australia.locations"),
-            TestSpec.of(astralia::animals, "australia.animals"),
-            TestSpec.of(astralia::states, "australia.states"));
+        Australia australia = faker.australia();
```
Comment (kingthorin) [inline src/test/java/net/datafaker/providers/base/AustraliaTest.java:10]: Ugh, sorry that was me

Comment (bodiam) [reply src/test/java/net/datafaker/providers/base/AustraliaTest.java:10]: I didn't check, I feared it was me :-)

**Review by kingthorin — APPROVED** [review]

**Pull request merged**